### DISCUSSION TITLE
Big format notes compatibility (books and larger structured content)

### DISCUSSION THREAD

**Discussion by NickG13** [standardnotes/forum#2642] [Suggestions]
I'm a writer and I'm considering importing all my Scrivener projects onto StandardNotes, but it doesn't have any kind of template neither any option to make this migration a good experience. The privacy feature is great, but I think this app needs some kind of structure inside the notes. Maybe a Table of Contents, a comment option and/or an editor extension for books, like Reedsy has. 

I know you guys doesn't take features request, but please take in consideration this one, since imho this is what SN lacks (even tho, I have payed for a 5 year license)

**Comment by JaspalSuri** [comment]
Hi @NickG13, thanks for the feedback! Hopefully someone from the community will provide an editor that provides whatever is needed to make book writing easier. 🙂
